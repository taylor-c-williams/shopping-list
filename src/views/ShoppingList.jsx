import React from 'react';
import Input from '../components/Input';
import List from '../components/List';

export default function ShoppingList() {
  return (
    <>
      <Input />
      <List />
    </>
  );
}
