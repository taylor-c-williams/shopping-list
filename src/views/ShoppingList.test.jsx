import React from 'react';
import ShoppingList from './ShoppingList';

it('should render the Shopping List view', () => {
  const { container } = <ShoppingList />;
  expect(container).toMatchSnapshot();
});
