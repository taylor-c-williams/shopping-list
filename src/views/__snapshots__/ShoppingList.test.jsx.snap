// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Shopping List view 1`] = `undefined`;
